package.json

```diff
@@ -36,6 +38,11 @@
     "@types/node": "^12.12.17",
     "@types/styled-components": "^5.0.1",
     "babel-loader": "^8.0.6",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-prettier": "^3.1.3",
```

Comment:
do we need this?

Reply:
Up to you, we can remove it if you like

Reply:
im fine just using prettier defaults. i dont think we need eslint ones as well.